utils/getInvoices/tests/getInvoices.maintenance.test.ts

```diff
@@ -6,108 +6,834 @@ import { ServiceType } from '@utils/constants'
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Maintenance,
+            amount: 10,
+            price: 10,
+            sum: 10,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = null, prevPayment = { invoice: [Electricity] }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Electricity,
+            price: 10,
+            amount: 10,
+            lastAmount: 0,
+            sum: 100,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Maintenance })
+      )
+    })
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 10 }, prevPayment = { invoice: [Maintenance] }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 10,
+      }
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Maintenance,
+            amount: 10,
+            price: 15,
+            sum: 15,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Maintenance,
+        amount: 10,
+        price: 10,
+        sum: 100,
+      })
+    })
+    it('should load when service = { rentPrice: 10 }, company = { totalArea: 10 }, prevPayment = { invoice: [Electricity] }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = {
+        totalArea: 10,
+      }
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Electricity,
+            price: 10,
+            amount: 10,
+            lastAmount: 0,
+            sum: 100,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Maintenance,
+        amount: 10,
+        price: 10,
+        sum: 100,
+      })
+    })
+    it('should NOT load when service = { rentPrice: 10 }, company = { discount }, prevPayment = { invoice: [Maintenance] }', () => {
```

Comment:
rename to `should NOT load when service = { rentPrice: 10 }, company = { ... }, prevPayment = { invoice: [Maintenance] }`